staging/src/k8s.io/component-base/cli/run.go

```diff
@@ -125,20 +147,20 @@ func run(cmd *cobra.Command) (logsInitialized bool, err error) {
 	case cmd.PersistentPreRun != nil:
 		pre := cmd.PersistentPreRun
 		cmd.PersistentPreRun = func(cmd *cobra.Command, args []string) {
-			logs.InitLogs()
+			logs.InitLogs(logs.FeatureGate(o.featureGate))
```

Comment:
component-base/cli doesn't use the FeatureGate itself. It just passes through to component-base/logs.InitLogs.

Reply:
I'm not sure about the `logs.FeatureGate` and `cli.FeatureGate` names: it is not obvious that these are function calls and not type casts. `WithFeatureGate`? gRPC uses `With*` for functional options, logr doesn't.